spec/rubocop/cop/ezcater/feature_flag_name_valid_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Ezcater::FeatureFlagNameValid, :config do
+  subject(:cop) { described_class.new(config) }
+
+  before { inspect_source(line) }
+  describe "assignment to a constant" do
+    context "with a valid feature flag name" do
+      let(:line) { %[MY_FLAG = "Foo::Bar"] }
```

Comment:
:8ball: It would be nice to see us testing a few more valid flag names here. Then our specs can serve as self documenting tests for what's a valid name.